pkg/interfaces/contracts/vault/ICompositeLiquidityRouter.sol

```diff
@@ -72,35 +76,39 @@ interface ICompositeLiquidityRouter {
      * @notice Remove proportional amounts of underlying from an ERC4626 pool, burning an exact pool token amount.
      * @dev An "ERC4626 pool" contains IERC4626 yield-bearing tokens (e.g., waDAI).
      * @param pool Address of the liquidity pool
+     * @param isWrappedToken An array indicating whether the output token is a wrapper or underlying token
      * @param exactBptAmountIn Exact amount of pool tokens provided
-     * @param minUnderlyingAmountsOut Minimum amounts of underlying tokens out, sorted in token registration order of
+     * @param minAmountsOut Minimum amounts of underlying tokens out, sorted in token registration order
      * wrapped tokens in the pool
      * @param wethIsEth If true, incoming ETH will be wrapped to WETH and outgoing WETH will be unwrapped to ETH
      * @param userData Additional (optional) data required for removing liquidity
-     * @return underlyingAmountsOut Actual amounts of tokens received, sorted in token registration order of wrapped
+     * @return amountsOut Actual amounts of tokens received, sorted in token registration order
      * tokens in the pool
      */
     function removeLiquidityProportionalFromERC4626Pool(
         address pool,
+        bool[] memory isWrappedToken,
         uint256 exactBptAmountIn,
-        uint256[] memory minUnderlyingAmountsOut,
+        uint256[] memory minAmountsOut,
         bool wethIsEth,
         bytes memory userData
-    ) external payable returns (uint256[] memory underlyingAmountsOut);
+    ) external payable returns (uint256[] memory amountsOut);
```

Comment:
Since we're not returning only underlying tokens, I believe it's confusing to don't return an array with the token addresses, so the user could validate if the router returned underlying or wrapped tokens, and which underlying it has returned.